python/libraries/bimvee/plotImu.py

```diff
@@ -60,6 +70,77 @@ def plotImu(inDict, **kwargs):
     axesMag.set_title('Mag (?)')
     axesMag.legend(['x', 'y', 'z'])
 
+#-----------------------------------------------------------------------------------------------------
+def plotImuDistribution(imuDict, unitIMU, fig_path, fig_name, fig_subtitle=None):
```

Comment:
unitIMU could be optional, by adding a default option in the definition line, e.g.:

def plotImuDistribution(imuDict, unitIMU='SI', fig_path, fig_name, fig_subtitle=None):

likewise, fig_path and fig_name could be optional - then it would test for their presence and if present it would execute the lines that save and close the figure.

Reply:
Commit e0be329 should solve this point.